thriftpy/parser/parser.py

```diff
@@ -399,6 +400,8 @@ def p_simple_base_type(p):  # noqa
         p[0] = TType.BOOL
     if p[1] == 'byte':
         p[0] = TType.BYTE
+    if p[1] == 'i8':
```

Comment:
This conditions should probably become `elif` so we don't continuously reevaluate `p[1]` after we find a match.

... but that would be better down in a separate change.